packages/component_code_gen/helpers/langchain_helpers.py

```diff
@@ -68,10 +67,11 @@ def get_llm():
     if config['openai_api_type'] == "azure":
         azure_config = config["azure"]
         llm = AzureChatOpenAI(deployment_name=azure_config['deployment_name'],
-                model_name=azure_config["model"], temperature=0, request_timeout=300)
+                              model_name=azure_config["model"], temperature=config["temperature"], request_timeout=300)
```

Comment:
I added this so I could experiment with temperature. Now each agent will generate slightly more creative responses, and the final judge agent can combine and correct.